src/components/Summary/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from 'react';
+import { ResumeContext } from '@app/App';
+
+import './index.scss';
+
+const Summary = () => {
+  const { basics } = useContext(ResumeContext);
+  const { summary } = basics;
+  return (
+    <div className="summary">
+      <div>
+        <div className="title">Summary</div>
+        <div className="content">{summary}</div>
+      </div>
```

Comment:
Try to follow same **BEM** convention here also:

```suggestion
        <div className="summary__title">Summary</div>
        <div className="summary__content">{summary}</div>
      </div>
```